### PR TITLE
Added controls scene & button on Main Menu.

### PR DESCRIPTION
**Pull request by bras1223**

### CODE REVIEW

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameControlsFXMLController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * @author PTS34A
+ */
+public class GameControlsFXMLController implements Initializable {
+
+    @FXML
+    private Button btnLogOut;
+    @FXML
+    private Button btnReturn;
+
+    private static final Logger LOGGER = Logger.getLogger(GameControlsFXMLController.class.getSimpleName());
```
Comment (TehMessiah) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/GameControlsFXMLController.java]: MAJOR Remove this unused "LOGGER" private field. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1068)

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/GameControlsFXMLController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    private Player currentPlayer;
```
Comment (TehMessiah) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/GameControlsFXMLController.java]: MAJOR Remove this unused "currentPlayer" private field. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1068)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 5 issues
* CRITICAL 3 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. CRITICAL [FXMLConstants.java#L27](https://github.com/PTS3-S34A/User-Interface/blob/2c25d2766fe01b46362064a6b17598ba890a494d/Soccar [UI]/src/nl/soccar/ui/fx/FXMLConstants.java#L27): Remove this hard-coded password. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2068)
1. CRITICAL [FXMLConstants.java#L28](https://github.com/PTS3-S34A/User-Interface/blob/2c25d2766fe01b46362064a6b17598ba890a494d/Soccar [UI]/src/nl/soccar/ui/fx/FXMLConstants.java#L28): Remove this hard-coded password. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2068)
1. CRITICAL [FXMLConstants.java#L30](https://github.com/PTS3-S34A/User-Interface/blob/2c25d2766fe01b46362064a6b17598ba890a494d/Soccar [UI]/src/nl/soccar/ui/fx/FXMLConstants.java#L30): Remove this hard-coded password. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2068)

**Comment by ghost** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. CRITICAL [FXMLConstants.java#L27](https://github.com/PTS3-S34A/User-Interface/blob/08bfa9098047c131ce7f3eead0bd23ce6c7b68a0/Soccar [UI]/src/nl/soccar/ui/fx/FXMLConstants.java#L27): Remove this hard-coded password. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2068)
1. CRITICAL [FXMLConstants.java#L28](https://github.com/PTS3-S34A/User-Interface/blob/08bfa9098047c131ce7f3eead0bd23ce6c7b68a0/Soccar [UI]/src/nl/soccar/ui/fx/FXMLConstants.java#L28): Remove this hard-coded password. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2068)
1. CRITICAL [FXMLConstants.java#L30](https://github.com/PTS3-S34A/User-Interface/blob/08bfa9098047c131ce7f3eead0bd23ce6c7b68a0/Soccar [UI]/src/nl/soccar/ui/fx/FXMLConstants.java#L30): Remove this hard-coded password. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2068)

**Review by Laugslander — APPROVED** [review]

**Pull request merged**